lib/core/notifications/models/notification.model.ts

```diff
@@ -18,7 +18,8 @@
 export enum NOTIFICATION_TYPE {
     INFO = 'info',
     WARN = 'warning',
-    ERROR = 'error'
+    ERROR = 'error',
+    RECURSIVE = 'recursive'
```

Comment:
what is recurrsive?

Reply:
I'm adding some documentation to better explain what these different types do. Recursive notifications are all those notifications that keep coming with the same message over and over, so instead of having 15 identical notifications you would only get one. (As in the case of pending requests that keep coming every time that component is initialized)